pinc/page_table.inc

```diff
@@ -348,20 +348,20 @@ function echo_page_table(
 
     echo "</tr>";
 
-    $avail_pages = [];
-
     $row_count = 0;
     foreach (fetch_page_table_data($project, $page_selector, $proofed_in_round) as $page_res) {
         $row_count += 1;
 
         $imagename = $page_res['image'];
+        $page_state = $page_res['state'];
 
         echo "<tr>";
 
         // --------------------------------------------
         // Selector
         if ($can_edit) {
-            echo "<td><input type='checkbox' name='selected_pages[$imagename]'></td>\n";
+            $avail_class = page_state_is_a($page_state, "page_avail_state") ? " availbox" : "";
+            echo "<td><input type='checkbox' class='selectorbox$avail_class' name='selected_pages[$imagename]'></td>\n";
```

Comment:
Slight preference for keeping the class name just the name and not space + name -- the HTML won't care if there's an errant space in the `class` attr if `$avail_class` is empty -- and this makes it clearer that it's an additional class and not modifying the existing class.

```suggestion
            $avail_class = page_state_is_a($page_state, "page_avail_state") ? "availbox" : "";
            echo "<td><input type='checkbox' class='selectorbox $avail_class' name='selected_pages[$imagename]'></td>\n";
```